TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/NoteDetailFragment.kt

```diff
@@ -33,20 +35,22 @@ class NoteDetailFragment : Fragment() {
     enum class PaintColors { Red, Blue, Green, Yellow, Purple }
 
     private lateinit var drawView: PenDrawView
+    private var deleted = false
 
     companion object {
         lateinit var mListener: OnFragmentInteractionListener
 
-        internal fun newInstance(note: Note) = NoteDetailFragment().apply {
+        internal fun newInstance(inode: INode, note: Note) = NoteDetailFragment().apply {
```

Comment:
haha this is why i started asking those questions, i wasn't sure which should really be tied to the fragment/used to apply the saved instance state

Reply:
might be worth looking into whether or not we want to use a viewmodel/saved instance state for data like this when we start implementing spanning